### PR TITLE
NIAD-1085: release 0 0 1 into main

### PR DESCRIPTION
**Pull request by bartek-sarul**
## Description

Please explain the changes you have made here.

## Jira Ticket

Please add a link to the related Jira ticket (if any).

## Checklist

These are items (excluding GitHub Checks) which should be confirmed before a branch is ready to merge.

- [ ] Acceptance Criteria met
- [ ] Commit messages are meaningful
- [ ] Manually tested
- [ ] Self-reviewed your code
- [ ] Code reviewed from two other developers
- [ ] If your pull request depends on any other, please link them in the description
- [ ] main branch is passing/green on CI
- [ ] Add/update any relevant documentation

### CODE REVIEW

File: mesh/README.md [hunk 1]
```diff
@@ -16,6 +16,18 @@ accordingly)
 For all commands `[mailbox_id]` is the id of the mailbox used for the request. The `MAILBOX_PASSWORD` environment
 variable must be correct for the given id.
 
+### Environment set up
+
+Invoke the desired `env.sh` file to load environment variables that will be used by the mesh client
```
Comment (matthewdoreilly) [inline mesh/README.md:21]:
```suggestion
Invoke the desired `env.sh` file to load environment variables that will be used by the mesh client.
```

File: src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueMultiMessageTest.java [hunk 2]
```diff
@@ -46,7 +46,7 @@ void setUp() {
 
     @Test
     void whenMeshInboundQueueMessageIsReceivedThenMessageIsHandled(SoftAssertions softly)
-            throws IOException, JMSException, JSONException {
+        throws IOException, JMSException, JSONException {
```
Comment (matthewdoreilly) [inline src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueMultiMessageTest.java:49]: I'd revert this change, this part of the method header should be indented so it's clear from the method body. Think this may be happening due to the new formatting rules we've all imported to IntelliJ

File: src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueMultiMessageTest.java [hunk 3]
```diff
@@ -68,7 +68,7 @@ private void assertGpOutboundQueueMessages(SoftAssertions softly) throws IOExcep
     }
 
     private void assertGpOutboundQueueMessages(SoftAssertions softly, Message message, Resource fhirMessage)
-            throws IOException, JMSException, JSONException {
+        throws IOException, JMSException, JSONException {
```
Comment (matthewdoreilly) [inline src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueMultiMessageTest.java:71]: I'd revert this change, this part of the method header should be indented so it's clear from the method body. Think this may be happening due to the new formatting rules we've all imported to IntelliJ

**Review by matthewdoreilly — APPROVED** [review]
Approved pending my last minor comments

**Review by andrewwa-kainos — APPROVED** [review]
